client-js/app/elements/labrad-registry.ts

```diff
@@ -111,20 +148,20 @@ export class LabradRegistry extends polymer.Base {
 
 
   cursorTraverse(event) {
-    if (this.selectedIdx === null || this.getOpenDialog() || this.$.search.focused) {
+    if (this.getSelectedIndex() === null || this.getOpenDialog() || this.$.search.focused) {
       return;
     }
 
     const item = this.$.combinedList.selectedItem;
+
+    // If we have an item, we want to traverse down.
```

Comment:
This comment seems a bit out of place right before doing something if we _don't_ have an item. I'd suggest just putting a doc string on the method to explain what it does, rather than inline comments.